.github/workflows/self-scheduled.yml

```diff
@@ -28,6 +28,9 @@ on:
       report_repo_id:
         required: true
         type: string
+      commit_sha:
+        required: false
```

Comment:
make it false, and below we use 

> commit_sha: ${{ inputs.commit_sha || github.sha }}

The caller needs to make sure it pass this whenever necessary